tools/hive/hadoop-2.6.0/scripts/bootstrap.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -x
+/usr/sbin/sshd
```

Comment:
Is there a reason that we have to start an SSH daemon?

Reply:
That is used by Hadoop.